src/parser/jobs/smn/modules/GeneralCDDowntime.ts

```diff
@@ -1,38 +1,50 @@
 import ACTIONS from 'data/ACTIONS'
 import {CooldownDowntime} from 'parser/core/modules/CooldownDowntime'
 
-const DEFAULT_ALLOWED_DOWNTIME = 4000
 // Although the trance cooldown is 55s, it is better in many cases
 // to hold for a 60s alignment, so some downtime per use must be allowed.
 const TRANCE_ALLOWED_DOWNTIME = 5000
 
 export default class GeneralCDDowntime extends CooldownDowntime {
-	defaultAllowedAverageDowntime = DEFAULT_ALLOWED_DOWNTIME
-	trackedCds = [
-		{
-			cooldowns: [ACTIONS.DREADWYRM_TRANCE, ACTIONS.FIREBIRD_TRANCE],
-			allowedAverageDowntime: TRANCE_ALLOWED_DOWNTIME,
-		},
-		{cooldowns: [
+	trackedCds = [ {
+		cooldowns: [
+			ACTIONS.DREADWYRM_TRANCE,
+			ACTIONS.FIREBIRD_TRANCE,
+		],
+		allowedAverageDowntime: TRANCE_ALLOWED_DOWNTIME,
+		firstUseOffset: 7500,
+	}, {
+		cooldowns: [
 			ACTIONS.ENERGY_DRAIN,
 			ACTIONS.ENERGY_SIPHON,
-		]},
-		{cooldowns: [
+		],
+		firstUseOffset: 2500,
+	}, {
+		cooldowns: [
 			ACTIONS.ASSAULT_I_AERIAL_SLASH,
 			ACTIONS.ASSAULT_I_EARTHEN_ARMOR,
 			ACTIONS.ASSAULT_I_CRIMSON_CYCLONE,
-		]},
-		{cooldowns: [
+		],
+		firstUseOffset: 1000,
+		// isAffectedBySpeed: true,
```

Comment:
Leftover from an experiment?  Can this be removed?

Reply:
It's something that's defined in a comment on CooldownGroup as something we might implement in the future if we get confident in our GCD length estimates.  It would apply to the other assault as well if we leave it on here.